scipy/optimize/_linprog.py

```diff
@@ -184,16 +184,16 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     ----------
     c : 1D array
         Coefficients of the linear objective function to be minimized.
-    A_ub : 1D array, optional
-        2D array which, when matrix-multiplied by ``x``, gives the values of
-        the upper-bound inequality constraints at ``x``.
+    A_ub : 2D array, optional
+        2D array such that ``A_ub`` @ ``x`` gives the values of the upper-bound
```

Comment:
I meant these ones 

![image](https://user-images.githubusercontent.com/1303842/46581691-599bd200-ca3d-11e8-85f3-3a823abf50d1.png)

Reply:
@ilayn I thought the convention is to mark variables as ``variable_name``? Looking at the picture I agree it looks odd. If you think it's worth rewriting I can do it tomorrow.

Reply:
I guess the relevant guidelines are:
https://numpydoc.readthedocs.io/en/latest/format.html
http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
From the numpy guidelines:

> Enclose variables in single backticks. 
 
So, technically, this should be: `A` @ `x`, right? To me  ``A @ x`` still looks better/readable and worth going back and changing.

Reply:
Yes since you are referring to a math expression rather than the individual variables involved, I guess `A @ x` would be clearer. But again, just a suggestion, I don't have a strong opinion.